frame/staking/src/lib.rs

```diff
@@ -1032,6 +1051,34 @@ decl_storage! {
 pub mod migrations {
 	use super::*;
 
+	pub mod v7 {
+		use super::*;
+
+		pub fn pre_migrate<T: Config>() -> Result<(), &'static str> {
```

Comment:
💯

Reply:
@joao-paulo-parity this is an example PR, and an example command that we want a bot to execute is: 

```
cargo run --release --features=try-runtime try-runtime live wss://rpc.polkadot.io -s polkadot
```